src/css/styles.css

```diff
@@ -1,8 +1,57 @@
-body {
+/* body {
 	font-family: Arial, sans-serif;
 	display: flex;
 	justify-content: center;
 	align-items: center;
 	height: 100vh;
 	margin: 0;
+} */
+.buttons {
+	margin: 10%;
```

Comment:
Nu recomand sa scriem margini in unitate de masura %, pune totul in px mia bine.